syntax = "proto3";
package example;

option cc_generic_services = true;

// The request message 
message EchoRequest {
  string message = 1;
}

// The response message 
message EchoResponse {
  string message = 1;
}

// The echo service definition.
service EchoService {
  rpc Echo (EchoRequest) returns (EchoResponse) {}
  rpc Hello (EchoRequest) returns (EchoResponse) {}
}
